src/TranslationProvider.tsx

```diff
@@ -22,6 +22,7 @@ interface TranslationProviderProps {
   locale: Locale;
   namespaces?: Namespaces;
   translations?: TranslationData;
+  onError?: (error: Error | string) => void;
```

Comment:
probably out of scope of this PR but since you're using a very generic term (Error)...

you don't want to close the door for doing this with other types of errors, so you want to either have different functions (passed in an object) or pass the type of error in the callback (preferred)

Reply:
Actually onError is calling only with a `string` so removed the `Error` from the interface.